### PR TITLE
Massive issue-slaying PR to address ALL THE THINGS (well, many of them)

### PR DESCRIPTION
**Pull request by csuwildcat**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -76,25 +100,30 @@ <h2 id="terminology"><a class="toc-anchor" href="#terminology">§</a> Terminolog
 <td style="text-align:left">Unique ID string and PKI metadata document format for describing the cryptographic keys and other fundamental PKI values linked to a unique, user-controlled, self-sovereign identifier in a target system (i.e. blockchain, distributed ledger).</td>
 </tr>
 <tr>
-<td style="text-align:left">Subject</td>
+<td style="text-align:left">Holed</td>
```
Comment (Fak3) [inline index.html:103]: typo: Holder

Comment (csuwildcat) [reply index.html:103]: This wasn't in my change set, and to change this, you'll need to do it in the markdown file. Can you submit a separate PR for this that changes the MD file?

File: spec/spec.md [hunk 2]
```diff
@@ -865,7 +871,10 @@ Descriptor Objects_ are composed as follows:
         ****MUST**** be an array consisting of one or more valid URI strings for
         the acceptable credential schemas. A common use of multiple entries in
         the `uri` array is when multiple versions of a credential schema exist
-        and you wish to express support for submission of more than one version.
+        and you wish to express support for submission of more than one version.
```
Comment (wyc) [inline spec/spec.md:874]: should remove first person voice if possible

File: spec/spec.md [hunk 3]
```diff
@@ -865,7 +871,10 @@ Descriptor Objects_ are composed as follows:
         ****MUST**** be an array consisting of one or more valid URI strings for
         the acceptable credential schemas. A common use of multiple entries in
         the `uri` array is when multiple versions of a credential schema exist
-        and you wish to express support for submission of more than one version.
+        and you wish to express support for submission of more than one version. 
+        This field allowing multiple URIs ****IS NOT**** intended to be used as
```
Comment (wyc) [inline spec/spec.md:875]: should ****IS NOT**** be a normalized term here?

File: spec/spec.md [hunk 4]
```diff
@@ -1223,16 +1243,12 @@ credentials within the target data structure.
     },
     {
       "@context": "https://www.w3.org/2018/credentials/v1",
-      "id": "https://eu.com/claims/DriversLicense",
-      "type": ["EUDriversLicense"],
```
Comment (wyc) [inline spec/spec.md:1227]: From vc-data-model:
> The value of the type property MUST be, or map to (through interpretation of the @context property), one or more URIs.

You've removed the Type. I think this makes it no longer a VC. Is that the intent?

File: spec/spec.md (hunk 2 above)
Comment (glcohen) [reply spec/spec.md:874]: `one wishes`

File: spec/spec.md (hunk 3 above)
Comment (csuwildcat) [reply spec/spec.md:875]: I just wanted to really emphasize it, but we don't have to if you don't think it's right to do that.

File: spec/spec.md (hunk 4 above)
Comment (csuwildcat) [reply spec/spec.md:1227]: Wasn't the intent, so I guess I need to make up some random string to stick in type?

Comment (wyc) [reply spec/spec.md:1227]: +1

Comment (brentzundel) [reply spec/spec.md:1227]: according to the VC spec, the first URI in the type field MUST be `VerifiableCredential` anyway

File: index.html (hunk 1 above)
Comment (csuwildcat) [reply index.html]: I went ahead and made this change in the PR feedback update

File: spec/spec.md (hunk 2 above)
Comment (brentzundel) [reply spec/spec.md]: `and a Verifier wishes`

**Comment by csuwildcat** [comment]
I addressed all of these in the update to the PR, so are we good to merge?

File: spec/spec.md (hunk 4 above)
Comment (wyc) [reply spec/spec.md:1227]: cc @csuwildcat i think we're good to go after this

File: spec/spec.md [hunk 5]
```diff
@@ -1223,16 +1243,13 @@ credentials within the target data structure.
...
+      "id": "https://business-standards.org/schemas/employment-history.json",
+      "type": ["GenericEmploymentCredential"],
```
Comment (brentzundel) [inline spec/spec.md:1247]:
```suggestion
      "type": ["VerifiableCredential", "GenericEmploymentCredential"],
```

File: spec/spec.md [hunk 6]
```diff
@@ -1327,26 +1344,22 @@ credentials within the target data structure.
...
-              {
-                "id": "2457913570",
-                "route": "DE-0753197542"
-              }
-            ]
-          }
+      "VC": {
+        "@context": "https://www.w3.org/2018/credentials/v1",
+        "id": "https://business-standards.org/schemas/employment-history.json",
+        "type": ["GenericEmploymentCredential"],
```
Comment (brentzundel) [inline spec/spec.md:1350]:
```suggestion
        "type": ["VerifiableCredential", "GenericEmploymentCredential"],
```

File: spec/spec.md [hunk 7]
```diff
@@ -36,7 +36,13 @@ requirements.
 To address these needs, this Presentation Exchange specification codifies the
 `Presentation Definition` data format Verifiers can use to articulate proof
 requirements, as well as the `Presentation Submission` data format Holders can
-use to submit proofs in accordance with them.
+use to submit proofs in accordance with them. The specification is designed to 
+be both credential format and transport envelope agnostic, meaning an implementer 
+can use JWTs, VCs, JWT-VCs, or any other credential format, and convey them 
+via OIDC, DID Comms, CHAPI, VPs, or any other transport envelope. The goal of
```
Comment (OR13) [inline spec/spec.md:42]:
```suggestion
via OIDC, DID Comms, CHAPI, or any other transport envelope. The goal of 
```

Comment (OR13) [reply spec/spec.md:42]: We are doing VPs over all those transports.

**Comment by csuwildcat** [comment]
I pulled in all the suggested changes.

**Pull request merged**